flutter/documentation/custom-tasks.md

```diff
@@ -9,13 +9,23 @@ This file describes how to configure the application to use non-default benchmar
 
 # Using custom tasks.pbtxt file
 
-Create file named `benchmarksConfigurations.json`
-and put it into root application directory.
-Add new paths to the file `tasks.pbtxt` in following format:
-
-`${Name of option}: ${path}`
-
-`tasks.pbtxt` file is a resource.
+When the app starts for the first time, it creates a file named `benchmarksConfigurations.json` 
+in the root application directory.
+Each entry in the `benchmarksConfigurations.json` is the name and location of a configuration file.
+You can have multiple configurations stored in the `benchmarksConfigurations.json` and switch 
+between them in the app settings. 
+The `benchmarksConfigurations.json` looks like this:
+```json
+{
+   "default":"https://raw.githubusercontent.com/mlcommons/mobile_models/main/v1_0/assets/tasks_v2.pbtxt",
+   "otherName":"app:///path/to/custom_tasks.pbtxt" 
```

Comment:
Only filename matters when specifying config file path.
See here: https://github.com/mlcommons/mobile_app_open/blob/master/flutter/lib/benchmark/benchmark.dart#L351-L353
App would throw out `app:///path/to/` or any other prefix and look specifically for `<app dir>/custom_tasks.pbtxt`

Reply:
There is a check for HTTP prefix some line above, so the app would just use the config file path as is in this case. I fixed it by parsing the path first some line under if it is not an HTTP link.